### ISSUE TITLE
Search results are grouped by page types

### ISSUE THREAD

**Issue by dannyparsons** [unicef/iogt#83]
When a front-end user searches the site, the search results are displayed in groups based on page types:

1. Surveys/polls/quizzes
2. Articles

i.e. as below

![image](https://user-images.githubusercontent.com/11214914/120347824-8559dd80-c2f4-11eb-9143-2f159e7b2592.png)

**Comment by ehmadzubair** [comment]
I'm working on this ticket.

**Comment by ehmadzubair** [comment]
@dannyparsons The surveys/polls features aren't done yet. If they are the only Page types we want to add other than Article, I think this Issue is blocked.

If however we want to add sections to the results (not sure about the its utility), I can add them. However, we'll have to redo this effort when surveys/polls are done.

Thoughts?

**Comment by ehmadzubair** [comment]
Blocked by #5 #6 #38

**Comment by mikowhy** [comment]
@ehmadzubair 
> I'm working on this ticket.

during yesterday planning @dannyparsons mentioned this can be a task for us. Have you already started work with it?

**Comment by ehmadzubair** [comment]
> @ehmadzubair
> 
> > I'm working on this ticket.
> 
> during yesterday planning @dannyparsons mentioned this can be a task for us. Have you already started work with it?

No, Feel free to take it up

**Comment by mikowhy** [comment]
PR -> https://github.com/unicef/iogt/pull/122

**Comment by mikowhy** [comment]
@ehmadzubair / @dannyparsons could you please take a look at my PR -> https://github.com/unicef/iogt/pull/122

Ps. It has been already sent to internal code review while @mwrobel-merixstudio is on his holidays

**Comment by mikowhy** [comment]
you can find unmerged commits from  https://github.com/unicef/iogt/pull/122 in new PR -> https://github.com/unicef/iogt/pull/237

**Comment by sheralim012** [comment]
@ChrisMarsh82 this is done and you can close this.

**Comment by geoo89** [comment]
Confirmed this is working.
